src/const/about.js

```diff
@@ -0,0 +1,26 @@
+import { FaAward } from "react-icons/fa6";
```

Comment:
Consider creating a single `const.js` file to centralize and manage all static data. This will improve maintainability, enhance readability, and ensure consistency across the codebase. The current size of the codebase, there's no need to create separate files for each component at this stage.